internal/pkg/cli/storage_init.go

```diff
@@ -0,0 +1,237 @@
+// Copyright Amazon.com, Inc or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+const (
+	dynamoDBStorageType = "DynamoDB"
+	s3StorageType       = "S3"
+)
+
+var storageTypes = []string{
+	dynamoDBStorageType,
+	s3StorageType,
+}
+
+var (
+	fmtStorageInitTypePrompt = "What " + color.Emphasize("type") + " of storage would you like to associate with %s?"
+	storageInitTypeHelp      = `The type of storage you'd like to add to your service. 
+	DynamoDB is a key-value and document database that delivers single-digit millisecond performance at any scale.
```

Comment:
We need to remove the prefixed extra space for this line otherwise it becomes awkwardly indented :p